uiowa/uiowa
# Allow webmasters to add expand/collapse all links to individual forms.

### Problem
All users cannot uncheck the "Display collapse/expand all details link" for Individual webforms:

![image](https://user-images.githubusercontent.com/1068523/129225038-082d684d-3b7b-4767-8377-dde34ff93ca9.png)

This doesn't look good on the form I'm building nor does it add much value: https://diversity.uiowa.edu/form/report-a-problem

### Proposed solution
Uncheck the global webform setting for this config and let people add it back to individual webforms if they want.

# How to test
- Sync a site.
- Create a webform with more than 1 details element.
- See that the expand/collapse all links do not appear.
- See that you can add that in the form settings if you'd like.
- Review the SLEW of other webform settings changes that happened on export... 😬
